### PR TITLE
 Added codechef online challenges

### PR DESCRIPTION
**Pull request by abhishekpaharva**
**Changes:**

Added codechef online challenges

### CODE REVIEW

File: code/online_challenges/src/codechef/CHDIGER/CHDIGER.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#include <bits/stdc++.h>
```
Comment (arnavb) [inline code/online_challenges/src/codechef/CHDIGER/CHDIGER.cpp]: #1915

File: code/online_challenges/src/codechef/CHDIGER/CHDIGER.cpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+using namespace std;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/CHDIGER/CHDIGER.cpp]: Don't use this.

File: code/online_challenges/src/codechef/CHDIGER/README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Problem Link:
+[CHDIGER](https://www.codechef.com/MARCH19B/problems/CHDIGER).
```
Comment (arnavb) [inline code/online_challenges/src/codechef/CHDIGER/README.md]: Add more details than just a link. Look at the READMEs for other coding challenges.

File: code/online_challenges/src/codechef/JAIN/JAIN.cpp [hunk 4]
```diff
@@ -0,0 +1,71 @@
+#include <bits/stdc++.h>
+
+#define fastio ios_base::sync_with_stdio(false); cin.tie(NULL);  cout.tie(NULL)
```
Comment (arnavb) [inline code/online_challenges/src/codechef/JAIN/JAIN.cpp]: Don't use these defines. Don't use a `register int`, a `std::size_t` is fine. Don't use random integer types, prefer `int` for  nearly all cases (use `long`/`long long` _only when necessary_).

File: code/online_challenges/src/codechef/JAIN/JAIN.cpp [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+            {
+                x = x | mp[a[i][j]];
+            }
+            if (s.find(x) == s.end())
+            {   
+                value.push_back(x);
+                count[x] = 1;
+                s.insert(x);
+            }
+            else count[x] = count[x] + 1;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/JAIN/JAIN.cpp]: Use compound assignment operators everywhere possible.

File: code/online_challenges/src/codechef/CHDIGER/README.md [hunk 6]
```diff
@@ -1,2 +1,19 @@
 # Problem Link:
 [CHDIGER](https://www.codechef.com/MARCH19B/problems/CHDIGER).
+
+# Discription
```
Comment (arnavb) [inline code/online_challenges/src/codechef/CHDIGER/README.md]: Fix spelling

**Comment by abhishekpaharva** [comment]
please review

File: code/online_challenges/src/codechef/CHDIGER/CHDIGER.cpp (hunk 2 above)
Comment (arnavb) [reply code/online_challenges/src/codechef/CHDIGER/CHDIGER.cpp]: Don't use this.

**Comment by arnavb** [comment]
@abhishekpaharva Why are you resolving my comments about `using namespace std`? I would like you to remove those lines.

**Comment by abhishekpaharva** [comment]
I have made all the changes as per required. please review it.

**Comment by arnavb** [comment]
djldk

**Pull request merged**